consensus/consensus-types/src/test_utils.rs

```diff
@@ -28,6 +28,7 @@ pub fn placeholder_certificate_for_block(
     certified_block_round: u64,
     certified_parent_block_id: HashValue,
     certified_parent_block_round: u64,
+    is_commit_guaranteed: bool,
 ) -> QuorumCert {
     // Assuming executed state to be Genesis state.
     let certified_block_state = ExecutedState::state_for_genesis();
```

Comment:
not related to this PR but we can create a vote_data at line 34 to avoid duplicating in line 56

Reply:
Done.

Reply:
done